### ISSUE TITLE
Remote EFB and mask

### ISSUE THREAD

**Issue by Sadawalebg** [flybywiresim/aircraft#8460]
Hello, i Know the team working in the remote function on a browser. So would it be possible to have a function that hides the tablets. Like hiding the shelves by clicking on the support, for example.

Thanks.

**Comment by tracernz** [comment]
I’m not sure that’s a good idea as there would be no way to change the settings if you cannot connect to it remotely for whatever reason.

**Comment by Sadawalebg** [comment]
no, the aim would be to be able to hide/unhide the tablet by pressing an
object in the cockpit, such as the tablet holder or the microphone, as in
the fenix.

Le ven. 2 févr. 2024 à 07:34, Michael Corcoran ***@***.***> a
écrit :

**Comment by TazerXI** [comment]
In the A380 there is now a basic OIT implimentation. I think an ability to hide the tablet, as well as the sidestick, is needed as they can block parts of the OIT display and menu navigation buttons.

![Image](https://github.com/user-attachments/assets/d655da90-2838-4f23-abf8-4f5243828699)
